src/commands/app/logs.js

```diff
@@ -46,11 +63,22 @@ Logs.flags = {
     description: 'Limit number of activations to fetch logs from ( 1-50 )',
     default: 1,
     char: 'l'
+  }),
+  action: flags.string({
+    description: 'Fetch logs for a specific action',
+    default: '',
+    char: 'a'
+  }),
+  strip: flags.boolean({
+    char: 'r',
+    description: 'strip timestamp information and output first line only',
+    default: false
+  }),
+  tail: flags.boolean({
```

Comment:
is tail our final choice? should we alias it with `watch`?
what is the point of adding a default for -a? if they want to get logs for an action or actions, they would need to specify it ...

Reply:
thx @purplecabbage  . Removed action default and added aliases.